Makefile

```
@@ -0,0 +1,23 @@
+.PHONY: install test test-solidity test-cli test-integration clean
+
+install:
+	cargo install --path crates/solidity && \
+	npm install && npm fund
+
+test: install test-integration test-cli test-solidity
+
+test-integration: install
+	cargo test --package revive-integration
+
+test-solidity: install
+	cargo test --package revive-solidity
+
+test-cli: install
+	npm run test:cli
+
+clean:
+	cargo clean && \
+	rm -rf node_modules && \
+	rm -rf crates/solidity/src/tests/cli-tests/artifacts && \
+	rm -f ~/.cargo/bin/zksolc && \
```

`$HOME/.cargo/` is just the default, we should use `cargo uninstall instead`

```suggestion
	cargo uninstall revive-solidity
```